migrations/server/mysql/0002_role_on_keys.up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `timestamp_keys` ADD COLUMN `role` VARCHAR(255) NOT NULL, DROP KEY `gun`, ADD UNIQUE KEY `gun_role` (`gun`, `role`);
```

Comment:
Good catch! I don't think we want to set it as `DEFAULT "timestamp"` as that would be a long term change for a point in time fix. Instead I'll add another line to the migration to set the value to "timestamp" for all existing records. This might create a small race condition between running the migration and updating the containers, but we can solve that by further manually running an update to set all records with a `NULL` role to "timestamp".